src/account/views/auth.py

```diff
@@ -76,18 +75,17 @@ def login(request):
 
     if branding_org:
         org_branding = OrganizationBranding.objects.filter(
-            org__slug=branding_org
+            org_slug=branding_org
```

Comment:
@Genza999 i don't think this change is right

its referencing `slug` through the `org` relationship on the `OrganizationBranding` class. So it should be `__`. Lets actually add a test for this i think.

```
@reversion.register
class OrganizationBranding(HandleRefModel):
    org = models.OneToOneField(Organization, on_delete=models.CASCADE)
```

Reply:
This has been fixed, thanks

Reply:
@Genza999 you need to add tests for that (and any/all changes)